web/src/compose_notifications.ts

```diff
@@ -245,6 +248,50 @@ export function notify_messages_outside_current_search(messages: Message[]): voi
     }
 }
 
+export function maybe_show_one_time_message_fading_banner(): void {
+    const is_faded = $(".recipient_row").hasClass("message-fade");
+    if (is_faded && onboarding_steps.ONE_TIME_NOTICES_TO_DISPLAY.has("messages_fading_banner")) {
+        let recipient;
+        let narrow_url;
+        if (compose_state.get_message_type() === "stream") {
+            const stream_name = compose_state.stream_name();
+            const topic_name = compose_state.topic();
+            const stream_id = compose_state.stream_id();
+            assert(topic_name !== undefined);
+            assert(stream_id !== undefined);
+
+            recipient = `#${stream_name} > ${topic_name}`;
```

Comment:
I wonder if we should consider making a variant of `web/templates/inline_decorated_stream_name.hbs`  called something like `web/templates/inline_decorated_stream_with_topic.hbs`, that could at least allow styling these more nicely in the future (e.g. with the correct privacy icon)? 

@alya thoughts?

Reply:
I think that'd be nice. We'll have to address that independently as `#${stream_name} > ${topic_name}` is used in few more banners.

Reply:
Sure, maybe let's open a "refactoring" label issue for it.

Reply:
I'll open an issue.